content/en/docs/reference/scheduling/config.md

```diff
@@ -255,7 +255,183 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
-## {{% heading "whatsnext" %}}
+### MultiPoint config
+
+There is an additional field in the profile config, `MultiPoint`, which allows 
+for easily enabling or disabling a plugin across several extension points. The intent 
+of `MultiPoint` config is to simplify the configuration needed for users and administrators 
+when using custom profiles.
+
+Consider a plugin, `MyPlugin`, which implements the `PreScore`, `Score`, `PreFilter`, 
+and `Filter` extension points. To enable `MyPlugin` for all its available extension 
+points, the profile config simply looks like:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: multipoint-scheduler
+    plugins:
+      multiPoint:
+        enabled:
+        - name: 'MyPlugin'
+```
+
+This would equate to manually enabling `MyPlugin` for all of its extension 
+points, like so:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: non-multipoint-scheduler
+    plugins:
+      preScore:
+        enabled:
+        - name: 'MyPlugin'
+      score:
+        enabled:
+        - name: 'MyPlugin'
+      preFilter:
+        enabled:
+        - name: 'MyPlugin'
+      filter:
+        enabled:
+        - name: 'MyPlugin'
+```
+
+Specific extension points can be excluded from `MultiPoint` expansion using 
+the `disabled` fields. This works with disabling default plugins, out-of-tree plugins, 
```

Comment:
nit: should it be field instead?

```suggestion
the `disabled` field. This works with disabling default plugins, out-of-tree plugins, 
```

Reply:
I think we should avoid talking about out-of-tree plugins in official k8s documentation.

Reply:
I don't disagree, but since this is different than how previous extension points treated out-of-tree plugins and this does apply to them, should those aspects be noted somewhere? Maybe part of the scheduling framework docs?